airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -43,42 +43,20 @@ def parse_dates(stream_slice):
     return start_date, end_date
 
 
-def get_date_params(start_date: str, time_zone=None, range_days: int = None, end_date: pendulum.datetime = None) -> Tuple[str, str]:
-    """
-    Returns `start_date` and `end_date` for the given stream_slice.
-    If (end_date - start_date) is a big date range (>= 1 month), it can take more than 2 hours to process all the records from the given slice.
-    After 2 hours next page tokens will be expired, finally resulting in page token expired error
-    Currently this method returns `start_date` and `end_date` with 15 days difference.
-    """
-
-    end_date = end_date or pendulum.yesterday(tz=time_zone)
-    start_date = pendulum.parse(start_date)
-    if start_date > pendulum.now():
-        return start_date.to_date_string(), start_date.add(days=1).to_date_string()
-    end_date = min(end_date, start_date.add(days=range_days))
-
-    # Fix issue #4806, start date should always be lower than end date.
-    if start_date.add(days=1).date() >= end_date.date():
-        return start_date.add(days=1).to_date_string(), start_date.add(days=2).to_date_string()
-    return start_date.add(days=1).to_date_string(), end_date.to_date_string()
-
-
 def chunk_date_range(
     start_date: str,
     conversion_window: int,
-    field: str,
     end_date: str = None,
     days_of_data_storage: int = None,
     range_days: int = None,
-    time_zone=None,
 ) -> Iterable[Optional[MutableMapping[str, any]]]:
     """
     Passing optional parameter end_date for testing
     Returns a list of the beginning and ending timestamps of each `range_days` between the start date and now.
     The return value is a list of dicts {'date': str} which can be used directly with the Slack API
     """
-    intervals = []
-    end_date = pendulum.parse(end_date) if end_date else pendulum.now()
+    yesterday = pendulum.yesterday()
```

Comment:
why do we no longer need to consider the timezone when calculating yesterday?

Reply:
good catch, I've missed it